### PR TITLE
Faq

### LINKED ISSUE #332: Add FAQ page
### Is your feature request related to a problem? Please describe.

FAQ page is must for this book finder website which is not present here.

### Describe the solution you'd like

- It will have similar UI as homepage
- It will have drop down answers feature
- It will be responsive

### Describe alternatives you've considered

 I want to add FAQ page.

Please assign this issue to me under JWOC.

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by S-ishita**
### Description
Added faq page.

### Checklist

- [x] I am making a proper pull request, not spam.
- [x] I've checked the issue list before deciding what to submit.

## Related Issues or Pull Requests

Closes #332 

## Add relevant screenshot or video (if any)

https://user-images.githubusercontent.com/104543751/224557830-f6f63160-a463-4d7a-9d83-5af949a53c72.mp4

### CODE REVIEW

File: faq.html [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+          <div class="description" style="text-align:left;">
+            <div class="containerf"> 
+               
+                <div class="question-container"> 
+                    <div class="question">Q:What is this website all about?</div> 
+                    <div class="answer">A: Book Finder is your go-to place for all kinds of book related searches.
+                    </div> 
+                    <hr>
+                    <div class="question">Q: What are the features of this website?</div> 
+                    <div class="answer">A: It helps you to read any time and anywhere.For more feature related information head to features page..
```
Comment (wasimreja) [inline faq.html:99]:
```suggestion
                    <div class="answer">A:  You can search books by title, author and ISBN number. For more feature related information head to features page.
```

File: faq.html [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+
+                    </div> 
+                    <hr>
+                    <div class="question">Q:  Do I need any special software to use it?</div>
```
Comment (wasimreja) [inline faq.html:103]:
```suggestion
```

File: faq.html [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+
+                    </div> 
+                    <hr>
+                    <div class="question">Q:  Do I need any special software to use it?</div> 
+                    <div class="answer">A: No, the only requirement is any web browser.
+
+                    </div> 
+                    <hr>
+                    <div class="question">Q:  Why should I prefer Book Finder over any other website?</div> 
+                    <div class="answer">A: Book Finder is a one stop solution for all kinds of book related searches making it preferable over other websites.
```
Comment (wasimreja) [inline faq.html:109]:
```suggestion
                    <div class="answer">A: Book Finder is open-source free search engine with over a million books making it preferable over other websites.
```

**Review by wasimreja — CHANGES_REQUESTED** [review]
-  faq page navbar is not updated.
- change font family of container.
- change faq illustration to [this.](https://assets.materialup.com/uploads/ba8ec2f8-f51b-4ff3-9586-a2a8b2d353bc/preview.jpg)

**Comment by S-ishita** [comment]
> * faq page navbar is not updated.
> * change font family of container.
> * change faq illustration to [this.](https://assets.materialup.com/uploads/ba8ec2f8-f51b-4ff3-9586-a2a8b2d353bc/preview.jpg)
> * change color of `+` to `#eba544`

faq navbar is updated..please check again

**Comment by wasimreja** [comment]
Screenshot 2023-03-12 at 23-39-29 Book Finder
Screenshot 2023-03-12 at 23-39-13 Book Finder

**Comment by S-ishita** [comment]
Done @wasimreja

**Comment by wasimreja** [comment]
don't change other pages style. 
Screenshot 2023-03-13 at 00-04-24 Book Finder
keep your nav like this just put faq before contact.
Screenshot 2023-03-12 at 23-39-29 Book Finder
otherwise it will look like this.
Screenshot 2023-03-13 at 00-14-08 Book Finder
you haven't changed the description as said. and remove _Do I need any special software to use it?_
`+` is looking too small. previous one was okay. i have said to change the color.

**Comment by S-ishita** [comment]
previous one was unicode its color cant be changed

**Comment by S-ishita** [comment]
> previous '+' was unicode its color cant be changed

Can you please tell if i keep it unicode or make it color code one?

**Comment by wasimreja** [comment]
keep it unicode and do the other changes

**Pull request closed without merging**